### ISSUE TITLE
panic on unwrap with empty string

### ISSUE THREAD

**Issue by anewton1998** [avnerbarr/radix-trie#1]
I am observing a panic from an unwrap when passing an empty string to `get_suffix_values`:

```
thread 'storage::mem::label_search::tests::test' panicked at /home/andy/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ab-radix-trie-0.2.0/src/lib.rs:55:43:
called `Option::unwrap()` on a `None` value
```

using this code
~~~rust
    #[test]
    fn test() {
        let mut trie: Trie<String> = Trie::new();
        trie.insert("romanus", None);
        trie.insert("romulus", None);
        trie.insert("rubens", None);
        trie.insert("ruber", None);
        trie.insert("rubicon", None);
        trie.insert("rubicundus", None);

        // get all suffixes under "rom" (flattened)
        let results = trie.get_suffixes_values("");
        println!("suffixes = {:?}", results);
    }
~~~

**Comment by avnerbarr** [comment]
Fixed in master. Published version 0.2.1

**Comment by avnerbarr** [comment]
fixed 0.2.1
